voluncheer/opportunityboard/views/search.py

```diff
@@ -60,7 +60,8 @@ def search(self):
         subsubcategory(default: None)
         Output: Opportunity list
         """
-        filtered_opportunity = Opportunity.objects.all()
+        filtered_opportunity = Opportunity.objects.filter(is_archive=False)
```

Comment:
Please update the docstring to reflect that this only searches un-archived opportunities.